tests/integration/pob_suite.go

```diff
@@ -289,6 +296,363 @@ func (s *POBIntegrationTestSuite) TestValidBids() {
 	})
 }
 
+// TestMultipleBids tests the execution of various valid auction bids in the same block. There are a few
+// invariants that are tested:
+//
+//  1. The order of transactions in a bundle is preserved when bids are valid.
+//  2. All transactions execute as expected.
+//  3. The balance of the escrow account should be updated correctly.
+//  4. Top of block bids will be included in block proposals before other transactions
+//     that are included in the same block.
+//  5. If there is a block that has multiple valid bids with timeouts that are sufficiently far apart,
+//     the bids should be executed respecting the highest bids until the timeout is reached.
+func (s *POBIntegrationTestSuite) TestMultipleBids() {
+	params := QueryBuilderParams(s.T(), s.chain)
+	escrowAddr := sdk.AccAddress(params.EscrowAccountAddress).String()
+
+	s.Run("broadcasting bids to two different validators (both should execute over several blocks) with same bid", func() {
+		// escrow account balance
+		escrowAcctBalanceBeforeBid := QueryAccountBalance(s.T(), s.chain, escrowAddr, params.ReserveFee.Denom)
+
+		// create bid 1
+		// bank-send msg
+		msg := Tx{
+			User:              s.user1,
+			Msgs:              []sdk.Msg{banktypes.NewMsgSend(s.user1.Address(), s.user2.Address(), sdk.NewCoins(sdk.NewCoin("stake", sdk.NewInt(100))))},
+			SequenceIncrement: 1,
+		}
+		// create bid1
+		bidAmt := params.ReserveFee
+		bid1, bundledTxs := CreateAuctionBidMsg(s.T(), context.Background(), s.user1, s.chain.(*cosmos.CosmosChain), bidAmt, []Tx{msg})
+
+		// create bid 2
+		msg2 := Tx{
+			User:              s.user2,
+			Msgs:              []sdk.Msg{banktypes.NewMsgSend(s.user2.Address(), s.user3.Address(), sdk.NewCoins(sdk.NewCoin("stake", sdk.NewInt(100))))},
+			SequenceIncrement: 1,
+		}
+		// create bid2 w/ higher bid than bid1
+		bid2, bundledTxs2 := CreateAuctionBidMsg(s.T(), context.Background(), s.user2, s.chain.(*cosmos.CosmosChain), bidAmt.Add(params.MinBidIncrement), []Tx{msg2})
+
+		// get chain height
+		height, err := s.chain.(*cosmos.CosmosChain).Height(context.Background())
+		require.NoError(s.T(), err)
+
+		// broadcast both bids
+		txs := BroadcastTxs(s.T(), context.Background(), s.chain.(*cosmos.CosmosChain), []Tx{
```

Comment:
this broadcasting to the same validator

Reply:
pls update the test case

Reply:
Discussed offline, functionality to be tested is that bids sent to same val can be committed over multiple blocks